### DISCUSSION TITLE
Make it available as a homebrew formula

### DISCUSSION THREAD

**Discussion by kishaningithub** [basecamp/kamal#631] [Ideas]
It would be great if this project can be made available as a homebrew formula

**Comment by acidtib** [comment]
@kishaningithub can you explain a bit more the request, to install kamal all you need to do is run `gem install kamal`

**Reply by kishaningithub** [reply]
But I want to `brew install kamal` 😊

I manage all softwares both cli and UI using homebrew, that way I can install, update and manage all softwares in a consistent manner

**Reply by acidtib** [reply]
heres a formula you can tap https://github.com/acidtib/homebrew-kamal

**Reply by kishaningithub** [reply]
Wow.. Thanks for sharing 😊 Would be great to have an official version from basecamp as well

**Reply by gregjotau** [reply]
Any plan to add this @djmb ? It would reduce the barrier to entry.
